site/index.html

```diff
@@ -16,76 +16,100 @@
+                        <i class="fas fa-search fa-2x"></i>
+                        <p>Reference</p>
+                    </a>
+                </div>
+
+                <div class="col-3 mr-auto">
+                    <a href="http://manual.ngs-lang.org" class="icon-link transition duration-150 ease-in-out">
+                        <i class="far fa-file-alt fa-2x"></i>
+                        <p>Manuals</p>
+                    </a>
+                </div>
+                <div class="col-1 mr-auto">
+                    <a href="http://wiki.ngs-lang.org" class="icon-link transition duration-150 ease-in-out" target="_blank">
+                        <i class="fas fa-book-open fa-2x"></i>
+                        <p>Wiki</p>
+                    </a>
+                </div>
+            </div>
+        </div>
+
+        <div class="col-auto ml-auto">
+            <div class="row justify-content-end">
+                <div class="col-2 ml-auto">
+                    <a href="http://source.ngs-lang.org" class="icon-link transition duration-150 ease-in-out" target="_blank">
+                        <i class="fab fa-github fa-2x"></i>
+                        <p>GitHub</p>
+                    </a>
+                </div>
+                <div class="col-2 ml-auto">
+                    <a href="http://videos.ngs-lang.org" class="icon-link transition duration-150 ease-in-out" target="_blank">
+                        <i class="fab fa-youtube fa-2x"></i>
+                        <p>YouTube</p>
+                    </a>
+                </div>
+                <div class="col-2 ml-auto">
+                    <a href="http://discord.ngs-lang.org" class="icon-link transition duration-150 ease-in-out" target="_blank">
+                        <i class="fab fa-discord fa-2x"></i>
+                        <p>Discord</p>
+                    </a>
+                </div>
+                <div class="col-1"></div>
+            </div>
         </div>
     </div>
-    <div class="row">
-        <div class="col-sm">
-            <h2>About NGS</h2>
+    <div class="row  justify-content-center">
+        <div class="col-mr-auto meeting-container pt-3 pb-3 pl-3 pr-3">
+            <span class="icon-link far fa-comments fa-2x pr-3"></span>
+            <span class="pb-3">Join us in <a href="http://discord.ngs-lang.org">Discord</a> for our weekly meeting, every Sunday at 16:00 GMT</span>
+        </div>
+    </div>
+    <div class="row pt-2 pb-2">
+        <div class="col-sm" style="text-align: justify">
+            <h2 class="text-left">About</h2>
 
             <p>
-                NGS is a unique combination of select features borrowed from other languages and original features.
-                NGS was built from the ground up focusing on daily systems engineering tasks.
+                NGS, that stands for Next Generation Shell, is a unique combination of select features borrowed from other languages and original features.
```

Comment:
`that stands` -> `short`

Reply:
@ilyash what do you mean?  like this:  `NGS, Next Generation Shell, is a...` ?